Kernel/Devices/SerialDevice.cpp

```diff
@@ -81,18 +39,20 @@ String SerialDevice::device_name() const
 
 UNMAP_AFTER_INIT void SerialDevice::initialize()
 {
-    set_interrupts(false);
-    set_baud(Baud38400);
-    set_line_control(None, One, EightBits);
```

Comment:
Why did we remove this? the defaults still need to be set

Reply:
The job of keeping track of default settings should be in the `TTY` class. I will add a `load_default_settings` method that we can call from the `SerialDevice` constructor and will generate the needed `set_baud`, `set_character_bits` etc. calls.